dexbot/cli.py

```diff
@@ -6,6 +6,7 @@
 import sys
 
 from dexbot.config import Config, DEFAULT_CONFIG_FILE
+from dexbot.cli_conf import *
```

Comment:
Could you fix this so that instead of using * import things that are required one by one.